project/BalboaClient.scala

```diff
@@ -34,7 +34,7 @@ object BalboaClientJMS {
 object BalboaClientDispatcher {
   lazy val settings: Seq[Setting[_]] = BalboaClient.settings ++ Seq(
     libraryDependencies <++= scalaVersion {libraries(_)},
-    ScoverageSbtPlugin.ScoverageKeys.coverageMinimum := 70
+    ScoverageSbtPlugin.ScoverageKeys.coverageMinimum := 64
```

Comment:
Yeah, it's a bummer. Two of the three classes in that package have 100% coverage, while the other one (`MetricLoggerToDispatcher`) has 0% because its behavior consists entirely of starting up a connection to ActiveMQ, which we obviously can't do during the unit tests.

The coverage went down because I removed some unnecessary code (which had tests).